srcs/object/object.c

```diff
@@ -10,24 +11,23 @@ t_shape	get_object_type(void *object)
 }
 
 // If there are two intersection points, return the distance to the closer one.
-static double	get_distance(t_vector ray, t_scene *scene, void *object)
+double	get_distance(const t_ray *ray, void *object)
 {
-	t_shape	type;
-	double	distance;
+	const t_shape	type = get_object_type(object);
+	double			distance;
 
-	type = get_object_type(object);
 	if (type == SPHERE)
-		distance = get_distance_to_sphere(ray, scene, (t_sphere *)object);
+		distance = get_distance_to_sphere(ray, (t_sphere *)object);
 	else if (type == PLANE)
-		distance = get_distance_to_plane(ray, scene, (t_plane *)object);
+		distance = get_distance_to_plane(ray, (t_plane *)object);
 	else if (type == CYLINDER)
-		distance = get_distance_to_cylinder(ray, scene, (t_cylinder *)object);
+		distance = get_distance_to_cylinder(ray, (t_cylinder *)object);
 	else
 		distance = NAN;
 	return (distance);
 }
 
```

Comment:
最初に初期化しておく形式で統一しますかねー？
```suggestion
	distance = NAN;
	if (type == SPHERE)
		distance = get_distance_to_sphere(ray, (t_sphere *)object);
	else if (type == PLANE)
		distance = get_distance_to_plane(ray, (t_plane *)object);
	else if (type == CYLINDER)
		distance = get_distance_to_cylinder(ray, (t_cylinder *)object);
	return (distance);
}

```

Reply:
ですねー。頭初期化で！